cannon/mipsevm/versions/detect_test.go

```diff
@@ -38,6 +38,15 @@ func TestDetectVersion(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, VersionMultiThreaded, version)
 	})
+
+	t.Run("MultiThreaded64Binary", func(t *testing.T) {
```

Comment:
How is this different to the `MultiThreadedBinary` version? I suspect we should use the changes in https://github.com/ethereum-optimism/optimism/pull/12050/files#diff-ee9a4960e5eebe708b684fa017a499bf514e47a8e090fb564f8d9273021444ef to just have hard coded examples of the state versions to test detection since we can't generate a state for all StateVersions from the same binary anymore.

Reply:
It's identical. I'll use the hardcoded examples instead (created a TODO for that).